### ISSUE TITLE
Aggregate fails on my computer.

### ISSUE THREAD

**Issue by fulmicoton** [gz/autoperf#1]
**Describe the bug**

Aggregate fails with the following error.
```rust
thread 'main' panicked at 'Can't decode line Failed converting '100,00' from str..', /home/paul/.cargo/registry/src/github.com-1ecc6299db9ec823/autoperf-0.9.2/src/aggregate.rs:204:25
note: Run with `RUST_BACKTRACE=1` environment varia
```

I work on linux and may have a partial french locale. (using comma as decimal separator.)

Here is a sample of one of the csv file in output.

```csv
# started on Thu May  2 10:04:18 2019

     0.250269405 ;CPU0 ;312998921 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250166318 ;100,00 ; ;
     0.250269405 ;CPU1 ;152645003 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250161346 ;100,00 ; ;
     0.250269405 ;CPU2 ;15724529 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250155188 ;100,00 ; ;
     0.250269405 ;CPU3 ;145700633 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250150265 ;100,00 ; ;
     0.250269405 ;CPU4 ;381577068 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250186825 ;100,00 ; ;
     0.250269405 ;CPU5 ;182555600 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250186021 ;100,00 ; ;
     0.250269405 ;CPU6 ;6933341 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250182045 ;100,00 ; ;
     0.250269405 ;CPU7 ;39118411 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;250178195 ;100,00 ; ;
     0.462766311 ;CPU0 ;7313894 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;212509412 ;100,00 ; ;
     0.462766311 ;CPU1 ;419256212 ; ;OFFCORE_REQUESTS_OUTSTANDING.DEMAND_RFO ;212511132 ;100,00 ; ;

```

**Machine (please complete the following information):**
 - Linux version: Linux paul-laptop 4.18.0-16-generic #17-Ubuntu SMP Fri Feb 8 00:06:57 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
 - perf version: perf version 4.18.20
 - autoperf version: autoperf 0.2
 - output of: breakpoint  cstate_core  i915      kprobe  power     tracepoint  uncore_cbox_0  uncore_cbox_2  uprobe
cpu         cstate_pkg   intel_pt  msr     software  uncore_arb  uncore_cbox_1  uncore_cbox_3

**Comment by gz** [comment]
Hi, Thanks a lot for submitting this bug report! It should be fixed in 0.9.3 (by setting LC_NUMERIC accordingly for the perf process).

**Comment by gz** [comment]
Released version 0.9.3. Please let me know if it doesn't fix your issue.
